scripts/create_clone

```diff
@@ -23,6 +23,9 @@ def parse_command_line(args):
                         help="(required) Specify a case to be cloned. If not a full pathname, "
                         "\nthe case to be cloned is assumed to be under then current working directory.")
 
+    parser.add_argument("--ensemble", default=1,
+                        help="clone an ensemble of cases, the case name argument must end in an integer.")
```

Comment:
Can you add a few more details here, such as:

The specified number of new cases will be created, each ending with a unique integer. The numbering will start with the value in the case name argument.

Reply:
Actually, it might be even better to give an example here instead of my suggestion. I like the example that you gave in the PR text, so maybe just add that to the help text here?

Reply:
I've updated the help section with your suggestion @billsacks

Reply:
Thanks, this looks good now.